htdocs/core/actions_massactions.inc.php

```diff
@@ -825,7 +825,7 @@
 	}
 
 	$arrayofinclusion=array();
-	foreach($listofobjectref as $tmppdf) $arrayofinclusion[]='^'.preg_quote(dol_sanitizeFileName($tmppdf).'.pdf','/').'$';
+	foreach($listofobjectref as $tmppdf) $arrayofinclusion[]='^'.preg_quote(dol_sanitizeFileName($tmppdf),'/').'[a-zA-Z0-9-_]*\.pdf$';
```

Comment:
If we do so, we will also include PDF that were added manually from the "Join files" tab because files have name
REF-xxx.pdf
When generated with ODX, filename is 
REF_nameofodx.pdf
So i try another fix